samples/bluetooth/scanning_while_connecting/src/main.c

```diff
@@ -437,6 +437,34 @@ int main(void)
 
 	hci_vs_sdc_central_acl_event_spacing_set(&event_spacing_params);
 
+	/** Set a small connection event length to leave more time for scanning.
+	 *
+	 * This can also be set at compile time by using the Kconfig option
+	 * CONFIG_BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT in the prj.conf file.
+	 */
+	sdc_hci_cmd_vs_event_length_set_t event_length_params = {
+		.event_length_us = 2000,
+	};
+
+	sdc_hci_cmd_vs_event_length_set(&event_length_params);
```

Comment:
Here we should use
```suggestion
	hci_vs_sdc_event_length_set(&event_length_params);
```
instead so that the code is portable to multicore devices